_posts/2020-02-14-running-a-great-socratic-seminar.md

```diff
@@ -0,0 +1,70 @@
+
+With links selected, tabs ordered and content highlighted, it is time to prepare yourself to lead discussion. Reading the material is the first step. Once you have read and highlighted the material, you should practice explaining it--this will uncover holes in your knowledge that were not previously apparent. The more you prep, the more enjoyable the event will be, both for you and the attendees. Ultimately, your goal is to have just enough of an understanding of the material so that you can contextualize the content in a way which avoids the attendees feeling completely lost. 
+
+You will have to balance this form of preparation with your own schedule, as there is always more to learn. Do your best to prepare, but do not over extend yourself. It is always good to remind yourself and attendees that the material is presented on a best effort basis--you are a conduit, not the source itself. Be open to making mistakes and encouraging the crowd to participate. Treat the event like the review process for open source software: stand on the shoulder of giants, don't box yourself into an island of "expertise" at the front of the room.
+
+In terms of presupposed knowledge, it is our recommendation that you communicate to your members that the basics of Bitcoin will not be explained at each event (e.g. what a hash function is). That said, when the ecosystem introduces novel concepts/schemes/mechanisms to the Bitcoin community, make sure to explain them. For example, if you are going to explain a new opcode, put the explanation in the context of preexisting opcodes and why it may or may not be an improvement to the protocol. That may require you to explain how opcodes work. Always keep in mind the perspective of the meetup members who come to each event--they play a key role in bringing life to the meetup. Find a way to quickly explain recurring themes in snippets so you do not bore people who regularly attend the events. 
+
+Once you have established context, discussion can begin. In some cases discussion will commence unprompted, although having some prepared discussion questions can force the attendees to think about the material in way that encourages participation. Not every topic requires discussion, so don't avoid subject matter you think is important because it may not be not be interesting for discussion. You're telling a story, some topics are minor plot devices, others are more central elements of "the conversation."
```

Comment:
not be ~not be~

Reply:
```suggestion
Once you have established context, discussion can begin. In some cases discussion will commence unprompted, although having some prepared discussion questions can force the attendees to think about the material in way that encourages participation. Not every topic requires discussion, so don't avoid subject matter you think is important because it may not be interesting for discussion. You're telling a story, some topics are minor plot devices, others are more central elements of "the conversation."
```